### PR TITLE
Fixes for building on ppc64le and 390x due to ensure-frozen-images

### PR DESCRIPTION
**Pull request by tophj-ibm**
Signed-off-by: Christopher Jones tophj@linux.vnet.ibm.com

This PR fixes build issues on power and z systems due to recent changes to ensure-frozen-images from #18315.

ping @tianon @jfrazelle

### CODE REVIEW

File: hack/make/.ensure-frozen-images [hunk 1]
```diff
@@ -1,21 +1,37 @@
...
-# on ARM we need images that work for the ARM architecture
-if [ "$DOCKER_ENGINE_OSARCH" = "linux/arm" ]; then
-	images=(
-		hypriot/armhf-busybox@ea0800bb83571c585c5652b53668e76b29c7c0eef719892f9d0a48607984f9e1
-		hypriot/armhf-hello-world@508c59a4f8b23c77bbcf43296c3f580873dc7eecb1f0d680cea3067e221fd4c2
-		hypriot/armhf-unshare@3f1db65f8bbabc743fd739cf7145a56c35b2a0979ae3174e9d79b7fa4b00fca1
-	)
-fi
+# image lists for different archs that should match what's in the Dockerfile (minus the explicit images IDs)
+case "$DOCKER_ENGINE_OSARCH" in
```
Comment (jfrazelle) [inline hack/make/.ensure-frozen-images]: pretty!!!!!!!

**Comment by jfrazelle** [comment]
thanks this is way better LGTM

**Comment by tianon** [comment]
LGTM

**Comment by jfrazelle** [comment]
actually if you are interested you can continue work on this :) https://github.com/jfrazelle/docker/commit/4e4a4fa40523f265061c2c4ccfd91df86f66d27b obviously in a different PR but thought I would mention it

File: hack/make/.ensure-frozen-images (hunk 1 above)
Comment (tophj-ibm) [inline hack/make/.ensure-frozen-images]: :heart_eyes:

**Comment by tophj-ibm** [comment]
yeah I saw you ping srini on it, I'll take a look too

**Comment by StefanScherer** [comment]
👍

**Pull request merged**